### DISCUSSION TITLE
chromVAR getMarkerFeatures() interpretation

### DISCUSSION THREAD

**Discussion by wuv21** [GreenleafLab/ArchR#1177] [Questions / Documentation] [answered]
Hi! Question regarding how to interpret the values from chromVAR with ArchR. I followed the ArchR guide and created a MotifMatrix from running `addDeviationsMatrix()`. Afterwards, I ran `getMarkerFeatures()` to find the differential motifs between two annotated clusters with `useSeqnames = "z"` and `useMatrix = "MotifMatrix"`. After looking at the returned object and filtering for motifs with FDR < 0.05, I noticed that there were a few motifs with a FDR < 0.05 but a log2FC of `NaN`. I'm a bit confused where the `NaN` is coming from, since I don't think this is a division by 0 or log(0) issue since it would return Inf or -Inf. Happy to provide more details if needed.

Thanks for providing ArchR to the community - have really enjoyed using it!

Thanks!
Vincent

**Comment by rcorces** [comment, marked as answer]
raised previously - https://github.com/GreenleafLab/ArchR/issues/78

log2FC values do not have meaning for chromVAR. I think this has already been removed from the output in the most up to date versions of ArchR. What version are you using?

**Reply by wuv21** [reply]
Thanks @rcorces for the reply and for linking the previous post. Sounds good, I will use the meandiff to look at direction of change. Using ArchR 1.0.1

**Reply by wuv21** [reply]
I also notice NaNs from running getMarkerFeatures() on the regular PeakMatrix (no chromVar) - would you recommend using the meandiff here as well?

**Reply by rcorces** [reply]
Thanks for bringing this up. The previous check that was implemented was not correct and the `Log2FC` assay was not properly being set to `NULL`. I've fixed this via https://github.com/GreenleafLab/ArchR/commit/48dccf361bab6d9191a4d9b61e1317acfa71167c on `release_1.0.2`

**Reply by rcorces** [reply]
@wuv21 - that doesnt sound right. I'm happy to take a look further if you are able to recapitulate what you see on the tutorial dataset. You can try upgrading to `release_1.0.2` in case you are bumping into something that has already been addressed on this development branch

**Reply by wuv21** [reply]
Looks like it's all good now after upgrading to release_1.0.2. Thanks so much!
